tools/dockerfile-add-scanner/main.go

```diff
@@ -0,0 +1,97 @@
+package main
```

Comment:
Makes sense to also add the Copyright and License lines in addition to addressing the revive complaint.

Reply:
Done.